apis/v1/httproute_types.go

```diff
@@ -263,6 +263,9 @@ type HTTPRouteRule struct {
 	// invalid, 50 percent of traffic must receive a 500. Implementations may
 	// choose how that 50 percent is determined.
 	//
+	// When a HTTPBackendRef refers to a Service that has no ready endpoints,
+	// implementations MAY return a 503 for requests to that backend instead.
```

Comment:
Although I agree with previous threads that we shouldn't require a specific status code here, it may be more accurate to use "SHOULD" here as a suggestion per RFC 2119.